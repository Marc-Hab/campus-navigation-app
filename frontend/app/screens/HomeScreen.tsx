import {StyleSheet, Text, View } from "react-native";
import StatusBar from "./components/StatusBar";


export default function HomeScreen(){
  return (

  <View
      style={styles.container}>

      <Text> Edit app/index.tsx to edit this screen.</Text>
      <View>
      </View>


    </View>
  );
}

const styles = StyleSheet.create({
  container: {
        flex: 1,
        flexDirection:'column',
        backgroundColor :'#fff',
        justifyContent: "center",
        alignItems: "center",
  },
});