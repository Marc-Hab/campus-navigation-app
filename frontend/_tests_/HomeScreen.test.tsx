import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from '../screens/HomeScreen';

test('renders correctly', () => {
  const tree = renderer.create(<Intro />).toJSON();
  expect(tree).toMatchSnapshot();
});