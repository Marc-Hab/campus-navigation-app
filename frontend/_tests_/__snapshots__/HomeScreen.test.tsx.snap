// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text>
     Edit app/index.tsx to edit this screen.
  </Text>
  <View />
</View>
`;
